lib/parser.js

```diff
@@ -220,7 +220,14 @@ module.exports = function (args, opts, y18n) {
     }
   }
 
+  // order of precedence:
+  // 1. command line arg
```

Comment:
I think I'd be tempted to have the arguments passed as CLI arguments to take precedence over environment variables.